2021/10/22 14:33:46 [error] 30#30: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal else echo Skip xdebug support:9003 (through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 192.168.12.1, server: symfony.loc, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.12.2:9000", host: "symfony.loc"
2021/10/22 14:33:46 [error] 30#30: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal else echo Skip xdebug support:9003 (through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 192.168.12.1, server: symfony.loc, request: "GET /_wdt/52295f HTTP/1.1", upstream: "fastcgi://192.168.12.2:9000", host: "symfony.loc", referrer: "http://symfony.loc/"
